runtime/near-vm-runner/src/metrics.rs

```diff
@@ -50,6 +74,15 @@ impl Metrics {
                 .observe(self.wasmtime_compilation_time.as_secs_f64());
             self.wasmtime_compilation_time = Duration::default();
         }
+        match self.compiled_contract_cache_hit.take() {
+            Some(true) => {
+                COMPILED_CONTRACT_CACHE_HIT.with_label_values(&[caller_context, shard_id]).inc();
+            }
+            Some(false) => {
+                COMPILED_CONTRACT_CACHE_MISS.with_label_values(&[caller_context, shard_id]).inc();
```

Comment:
It is better to measure the number of accesses and then compute the amount of misses from the ratio of hits to the accesses.

Reply:
Done.